docarray/array/storage/weaviate/backend.py

```diff
@@ -322,3 +329,8 @@ def _get_storage_infos(self) -> Dict:
             'Serialization Protocol': self._config.serialize_config.get('protocol'),
             **storage_infos,
         }
+
+    def flush_weaviate_batch(self) -> None:
+        """Flush existing objects in Weaviate Batch to the Weaviate Backend Server"""
```

Comment:
Without flushing, the objects will be in-memory batch of client-side? or backend server side?

Reply:
In memory of client-side.